utils_cv/action_recognition/model.py

```
@@ -357,7 +303,9 @@ def train_an_epoch(
                         loss = loss / grad_steps
 
                         if mixed_prec and AMP_AVAILABLE:
```

Same as above, should only probably check for mixed_prec and not silently ignore if AMP isn't available.

---

Marked as "hooray" but don't see change